sympy/algebras/quaternion.py

```diff
@@ -898,8 +900,8 @@ def angle(self):
         Explanation
         ===========
 
-        Given a quaternion ``q = a + b*i + c*j + d*k`` where a, b, c and d
-        are real numbers and returns angle of the quaternion given by
+        Given a quaternion $q = a + b*i + c*j + d*k$ where a, b, c and d
```

Comment:
I would remove the asterisks when inside LaTeX equations (between currency/dollar-sign symbols), and I would keep the asterisks when in "code / monospaced typewriter text style" (between backticks). Anyway, it's a matter of style.